components/brave_wallet_ui/components/desktop/side-nav-button/style.ts

```diff
@@ -25,7 +25,7 @@ export const ButtonText = styled.span<Partial<StyleProps>>`
 	font-weight: 600;
   letter-spacing: 0.02em;
 	color: ${(p) =>
-    p.isSelected ? '#ffffff' : `${p.theme.color.text}`};
+    p.isSelected ? '#ffffff' : `${p.theme.color.text02}`};
```

Comment:
`` `${p.theme.color.text02}` `` is exactly the same as just stating `p.theme.color.text02`

Reply:
fixed this!